modules/iam-assumable-role-with-oidc/variables.tf

```
@@ -105,3 +105,15 @@ variable "allow_self_assume_role" {
   type        = bool
   default     = false
 }
+
+variable "create_custom_role_trust_policy" {
+  description = "Whether to create a custom role trust policy to IAM role or not."
+  type        = bool
+  default     = false
+}
+
+variable "custom_role_trust_policy" {
```

```suggestion
variable "assume_role_policy" {
```

Description `Custom policy that grants an entity permission to assume the role. A policy will be created if a custom policy is not provided`

Default value should be `null`

---

Hello @bryantbiggs, changes done as per your peer review.
Thanks a lot for your precious feedback.
Let me know if there are still adaptations to be done.